src/components/BugsTable/index.jsx

```diff
@@ -0,0 +1,238 @@
+import { Component } from 'react';
+import { withStyles } from 'material-ui/styles';
+import Table, {
+  TableBody,
+  TableHead,
+  TableRow,
+  TableSortLabel,
+  TablePagination,
+  TableCell,
+} from 'material-ui/Table';
+import PROJECTS_PAGE_SIZE from '../../utils/constants';
+
+const columns = [
+  { label: 'Project' },
```

Comment:
We don't need this to be an object anymore since we're only dealing with one field property. Switch to an array of strings.